src/PreDraft/RegisteredPlayers.js

```diff
@@ -26,37 +25,33 @@ class RegisteredPlayers extends Component {
     this.subscription && this.subscription.end()
   }
 
-  render () {
+  render() {
+    const { me } = this.props
+    const playersList = this.state.data.map(user => {
+      if (user === me) {
+        return <EditTableContent key={user} user={user} me={me} />
+      } else {
+        return <PlayerTableContent key={user} user={user} me={me} />
```

Comment:
this if statement should use a ? operator like it did before and the check will be
user.id === me._key

or it will be close to that, depends on heimerdinger data

Reply:
Yeah, that's what it's currently doing. I destructured out the me,  and it's checking it against the key(I tested and I didn't need to put _key for some reason after checking the props in react dev tools). As far as moving it out of return statement, I just feel things are a little easier if within the jsx logic is left to a minimum. Then it reads a little more like html. As far as the ternary goes, that was just a personal decision because I like writing if else versus ternaries. So that of course can be changed.

Reply:
the user object might be different from heimerdinger in future.  I was just guessing what it will change to.